apps/frontend/src/app/pages/company/create-profile/create-profile.component.html

```diff
@@ -0,0 +1,366 @@
+<form class="container px-5 mx-auto min-h-screen" [formGroup]="form">
+  <div class="md:flex gap-36 pt-8">
+    @if(step === 1){
```

Comment:
Here is how it is in the screen with width 1536px:
![image](https://github.com/Venture-Miner/wagademy/assets/90660988/db6fe432-3249-403e-81b2-a79dcd50551f)
How it would be according to figma:
![image](https://github.com/Venture-Miner/wagademy/assets/90660988/d253cbd2-64c3-4425-8dad-cef40c8b484a)
This happens also in others screen sizes, is missing this spaces:
![image](https://github.com/Venture-Miner/wagademy/assets/90660988/7b2484c8-7b82-46fb-867a-d9dce13cb743)

And after the small screen, the screen for cellphones is like this:
![image](https://github.com/Venture-Miner/wagademy/assets/90660988/57a0936d-d74d-4702-bb18-dab9649bf8c6)

Reply:
Regarding the step layout, it works and is filled in according to the daisyUi step component: https://daisyui.com/components/steps/

And the side spacing is the same as the navbar: px-5